python/src/aiconfig/editor/client/src/components/AIConfigEditor.tsx

```diff
@@ -878,7 +885,10 @@ export default function EditorContainer({
               <Button
                 leftIcon={<IconDeviceFloppy />}
                 loading={isSaving}
-                onClick={onSave}
+                onClick={() => {
+                  onSave();
+                  logEvent?.("SAVE_BUTTON_CLICKED");
```

Comment:
Edit: Oh nm I see that the `data` field is optional: `data?: LogEventData`. Makes sense

** Old **
Do we need to pass in empty event data here?

```
logEvent?.("SAVE_BUTTON_CLICKED", {})
```

Reply:
Ok so in general how FB did logging, we'd have a field for UI component/surface, and then another field for EVENT (generated, clicked, etc).

But yea not a blocker, just FYI, this is good for now

Reply:
thanks for the insight!